workspaces/config/lib/definitions/definitions.js

```diff
@@ -2012,11 +2012,14 @@ const definitions = {
       If you ask npm to install a package and don't tell it a specific version,
       then it will install the specified tag.
 
-      Also the tag that is added to the package@version specified by the \`npm
-      tag\` command, if no explicit tag is given.
+      Also the tag that is added to the package@version specified by the 
+      \`npm dist-tag add\` command, if no explicit tag is given.
 
       When used by the \`npm diff\` command, this is the tag used to fetch the
       tarball that will be compared with the local files by default.
+      
+      If used in the \`npm publish\` command, this is the dist tag that will be 
+      added to the package submitted to the registry.
```

Comment:
Remove "dist"

Reply:
Makes sense, updated in 377c5b3a7437126177f09324d675b5b51ab11e15 thx